### ISSUE TITLE
Add help text for existing umbrella selection

### ISSUE THREAD

**Issue by aleksisalonen** [City-of-Helsinki/linkedevents-ui#516]
If an existing umbrella event has subevents, removing the umbrella-status of said event becomes impossible.

Let's tell the user this in the help text:
"Jos haluat tehdä tästä tavallisen tapahtuman, poista ensin viittaukset sen alatapahtumista."
"To change the umbrella status, remove all existing references from it's sub events first."

**Comment by Rikuoja** [comment]
Yeah, currently it seems the help text is "Alatapahtuma ei voi olla kattotapahtuma", which makes no sense when we are editing an umbrella event?
